app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -488,6 +492,10 @@ class MEJSPlayer {
     if (!this.player.paused) {
       this.player.pause();
     }
+
+    // Unbind custom resize event handler
+    this.player.globalUnbind('resize', this.customGlobalResizeCallback)
```

Comment:
I don't think this is necessary, since remove() will remove all bound event handlers too.

Reply:
Nevermind. I see that globalbind actually binds to the document, not the player.

Reply:
This was to replicate what was going on here (https://github.com/mediaelement/mediaelement/blob/26ec30053d3fd59c973fec0164989bda984bee21/src/js/player.js#L2023) in mediaelement's `remove()` since we're binding something new to it now.